mesonbuild/dependencies/base.py

```diff
@@ -24,6 +24,7 @@
 import textwrap
 import platform
 import typing
+import copy
```

Comment:
__[Flake8]__ [F811] redefinition of unused 'copy' from line 17 ([link](https://sider.review/gh/repos/19784232/pulls/5953#issue-4627264))

<sub>posted by Sider</sub>